### PR TITLE
fix qemu CVE-2020-15863

### PR DESCRIPTION
**Pull request by nicogbg**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES**
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/...

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

**Comment by lpereira** [comment]
Hm, the bot should have applied the "Security" tag automatically. Let me see what's going on here.

**Review by jslobodzian — APPROVED** [review]

**Comment by PawelWMS** [comment]
@nicogbg: please fill in the summary, checkboxes, change log, etc. in the PRs description. We should be properly commenting each change we're doing.

File: SPECS/qemu-kvm/qemu-kvm.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary:	QEMU is a machine emulator and virtualizer
 Name:		qemu-kvm
 Version:	4.2.0
```
Comment (PawelWMS) [inline SPECS/qemu-kvm/qemu-kvm.spec:3]: Nit: since you're already here, could you replace all tabs with spaces and align these fields?

**Review by PawelWMS — DISMISSED** [review]

**Review by lpereira — APPROVED** [review]
Looked at more context in the Qemu source for this patch. Looks good, although since this won't acknowledge this buffer descriptor, there's a chance that the (buggy) guest will be stuck; it at least stops memory from being read outside the bounds.

Doesn't seem to be much of a problem since this hardware model is rarely used in most VMs anyway.

**Pull request merged**